sound/soc/sof/topology.c

```diff
@@ -1746,6 +1752,15 @@ static int sof_widget_load_pipeline(struct snd_soc_component *scomp, int index,
 		goto err;
 	}
 
+	ret = sof_parse_tokens(scomp, swidget, pipeline_tokens,
+			       ARRAY_SIZE(pipeline_tokens), private->array,
+			       le32_to_cpu(private->size));
+	if (ret != 0) {
+		dev_err(scomp->dev, "error: parse pipeline tokens failed %d\n",
```

Comment:
this error message looks exactly like the one above, it makes it more difficult to identify the exact error